azure-functions/acbs-function/mappings/facility/facility-investor.js

```diff
@@ -15,12 +15,14 @@ const facilityInvestor = (deal, facility) => {
     guaranteeCommencementDate,
     guaranteeExpiryDate,
     effectiveDate,
-  } = facility.tfm.facilityGuaranteeDates;
+  } = facility.tfm.facilityGuaranteeDates
+    ? facility.tfm.facilityGuaranteeDates
+    : '';
 
   return {
-    facilityIdentifier: facility.ukefFacilityID
-      ? facility.ukefFacilityID.padStart(10, 0)
-      : facility.facilitySnapshot.ukefFacilityId.padStart(10, 0),
+    facilityIdentifier: facility.facilitySnapshot.ukefFacilityId
+      ? facility.facilitySnapshot.ukefFacilityId.padStart(10, 0)
+      : facility.facilitySnapshot.ukefFacilityID.padStart(10, 0),
```

Comment:
Same as above, future improvement can use the facility mappings used in mapSubmittedDeal, so then don't have ukefFacilityId/ukefFacilityID differences

Reply:
Will do